metricflow/aggregation_properties.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
```

Comment:
what if we had like a `metricflow/model/properties/` directory that stores these? I'm not aware of the best practices, but personally I like how we have most things organized in their respective directories rather than being in the root directory.

Reply:
I think that'd be fine, although I pulled these to root because I think of them as being on the level of specs and suchlike. To be honest I thought about moving those into a sub-directory as well, but decided not to get too carried away, so I'm open to some simple file move reorganization PRs.